Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdaterContentStrategy.cs

```diff
@@ -44,8 +44,8 @@ public void Update(CancellationToken cancellationToken)
             DebugLogger.Log("Updating with content strategy.");
 
             var commandFactory = new Commands.AppUpdaterCommandFactory();
-            var geolocateCommand = commandFactory.CreateGeolocateCommand();
 
+            var geolocateCommand = commandFactory.CreateGeolocateCommand();
```

Comment:
I know this might not be your code, but please replace var with a type.

Reply:
done